### PR TITLE
43172 : fix loadmore button display & shared contacts filtering

### PR DESCRIPTION
**Pull request by ahamdi**
This PR fixes two problems : 
 - loadMore button display : the button is displayed even if there are no more contacts to display
 - Shared filtering by Type : Filtering rooms by type (room, people, etc ..) is stored in browser storage, thus functions were modified to avoid to filter by room type when invoked from the Chat Drawer

### CODE REVIEW

File: application/package-lock.json [hunk 1]
```diff
@@ -24,6 +24,13 @@
         "webpack-merge": "^4.1.3"
       }
     },
+    "node_modules/@types/json-schema": {
```
Comment (hbenali) [inline application/package-lock.json:27]: Is it necessary to add changes of `package-lock.json` file?

Comment (ahamdi) [reply application/package-lock.json:27]: Yes , npm run watch didn't work as all dependency were outdated. I had to run npm update -f to update all packages with newer versions compatible with the upgraded NPM packages

Comment (ahamdi) [reply application/package-lock.json:27]: The problem is simple to reproduce, just download the develop branch, build it and then run : npm run watch
You will get this kind of exceptions : 
`Error: No version of chokidar is available. Tried chokidar@2 and chokidar@3.
You could try to manually install any chokidar version.
chokidar@3: Error: Cannot find module 'chokidar'`

Comment (hbenali) [reply application/package-lock.json:27]: Thanks for the explanation 😊

**Review by hbenali — APPROVED** [review]

**Pull request merged**